chia/types/peer_info.py

```diff
@@ -43,10 +46,12 @@ def port(self) -> uint16:
     # Functions related to peer bucketing in new/tried tables.
     def get_key(self) -> bytes:
         if self.ip.is_v4:
-            key = ipaddress.IPv6Address(int(ipaddress.IPv6Address("2002::")) | (int(self.ip) << 80)).packed
+            # key = ipaddress.IPv6Address(ipv6_2002_int | (int(self.ip) << 80)).packed
+            # below is a micro-optimization of above.  it avoids one creation of an ip address object.
+            key = (ipv6_2002_int | (int(self.ip) << 80)).to_bytes(16, byteorder="big")
```

Comment:
is `.to_bytes()` better than `.packed`?
I suppose it might be a bit clearer.

Reply:
The point is to not create the ip address object at all.  That is the object that has `.packed`.